build.rs

```diff
@@ -5,10 +5,17 @@
 use std::env;
 use std::process::{Command, Stdio};
 
+
+fn find_make() -> String {
+    Command::new("gmake").status()
+                         .map(|_| "gmake".to_string())
+                         .unwrap_or("make".to_string())
+}
+
 fn main() {
     let out_dir = env::var("OUT_DIR").unwrap();
     let target = env::var("TARGET").unwrap();
-    let result = Command::new("make")
+    let result = Command::new(env::var("MAKE").unwrap_or_else(find_make()))
```

Comment:
nit again: you're still calling the function, remove the two parens because it's not building now :P